src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookTests.java

```diff
@@ -1,127 +1,37 @@
 package com.wikia.webdriver.testcases.facebooktests;
 
-import com.wikia.webdriver.common.core.annotations.RelatedIssue;
-import com.wikia.webdriver.common.core.configuration.Configuration;
-import com.wikia.webdriver.common.core.helpers.User;
-import com.wikia.webdriver.common.driverprovider.UseUnstablePageLoadStrategy;
-import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.core.XMLReader;
+import com.wikia.webdriver.common.core.api.GraphApi;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
-import com.wikia.webdriver.common.users.FacebookTestUser;
-import com.wikia.webdriver.common.users.TestUser;
-import com.wikia.webdriver.pageobjectsfactory.componentobject.dropdowncomponentobject.DropDownComponentObject;
 import com.wikia.webdriver.pageobjectsfactory.componentobject.modalwindows.FacebookSignupModalComponentObject;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.facebook.FacebookMainPageObject;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.facebook.FacebookUserPageObject;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.facebook.RemoveFacebookPageObject;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.signup.AlmostTherePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.facebook.FacebookSettingsPageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.signup.SignUpPageObject;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.special.preferences.PreferencesPageObject;
-
-import org.joda.time.DateTime;
 import org.testng.annotations.Test;
+import java.util.HashMap;
 
 @Test(groups = {"auth-facebook"})
 public class FacebookTests extends NewTestTemplate {
 
-  Credentials credentials = Configuration.getCredentials();
-
-  TestUser user = FacebookTestUser.getUser();
+  String wikiaProductionAppId = XMLReader.getValue("facebook.prod.appId");
 
-  /**
-   * <ol>
-   *   <li>Log in to facebook</li>
-   *   <li>Click facebook login on signup page</li>
-   *   <li>Deny permission to user's facebook email address</li>
-   *   <li>>manually enter email address and create account</li>
-   *   <li>confirm account and login</li>
-   *   <li>Verify user can login via facebook</li>
-   * </ol>
-   */
   @Test(groups = "Facebook_userCanSignUpViaFacebook")
-  @UseUnstablePageLoadStrategy
-  @RelatedIssue(issueID = "SOC-2432", comment = "Test manually")
   public void userCanSignUpViaFacebook() {
-    new RemoveFacebookPageObject(driver).removeWikiaApps(user.getEmail(), user.getPassword());
+    GraphApi api = new GraphApi();
+    HashMap<String, String> test_user = api.createFacebookTestUser(wikiaProductionAppId);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00117)